octodns/source/base.py

```diff
@@ -9,6 +9,7 @@
 class BaseSource(object):
 
     SUPPORTS_MULTIVALUE_PTR = False
+    SUPPORTS_POOL_VALUE_STATUS = False
```

Comment:
I can't say if `BaseSource` is the right place for the flags or `BaseProvider`. I don't exactly see the implications of one vs the other.

Reply:
Iirc the thinking was that you could implement a source (not provider) that supported any of these advanced things  and thus the flags belong on the source. I don't think there's anything like that in practice, but as a thought expiriment imagine `YamlSource` rather than `YamlProvider` where it doesn't support being a target, but does support configuring values with whatever flags these are. 

Wouldn't be surprised in the slightest if there were shortcomings/issues if someone tried to do something of that sort since nothing has as of yet, but I believe that was the thinking.